src/main/java/assets/css/custom.css

```diff
@@ -1,3 +1,2 @@
 /* Add additional stylesheets below
--------------------------------------------------- */
-
+-------------------------------------------------- */
```

Comment:
There was actually no change made there. I accidentally modified this file and then deleted the contents, hence why it shows as if there was actually something of value. I ended up using AcesUpFinal.css instead. @mihaidan